django_admin_listfilter_dropdown/templates/admin/change_list.html

```diff
@@ -0,0 +1,52 @@
+{% extends "admin/change_list.html" %}
+{% load staticfiles %}
+
+{% block extrahead %}
+    <script type="text/javascript" src="{% static 'admin/js/vendor/jquery/jquery.js' %}"></script>
```

Comment:
I am looking to find a way to inject these assets into the ModelAdmin's Media.

Requirements:
- after jquery.js
- before jquery.init.js (appears to apply jQuery's noConflict which seems to cause errors in the select2 lib, Django Admin also applies the three libs in this order on the admin model detail page.

Reply:
Alternative would be to update the installation instructions to ask the developer to add this to any ModelAdmin that uses the select2 filter:

```
    class Media:
        js = (
            'admin/js/vendor/jquery/jquery.js',
            'admin/js/vendor/select2/select2.full.js',
            'admin/js/jquery.init.js',
            'admin/js/autocomplete.js',
        )

        css = {
            'screen': (
                'admin/css/vendor/select2/select.css',
                'admin/css/autocomplete.css',
            ),
        }
```